src/server/lib/server.ml

```diff
@@ -0,0 +1,103 @@
+(*
+  This file is part of scilla.
+
+  Copyright (c) 2018 - present Zilliqa Research Pvt. Ltd.
+
+  scilla is free software: you can redistribute it and/or modify it under the
+  terms of the GNU General Public License as published by the Free Software
+  Foundation, either version 3 of the License, or (at your option) any later
+  version.
+
+  scilla is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+  A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License along with
+  scilla.  If not, see <http://www.gnu.org/licenses/>.
+*)
+
+open Core
+open DebugMessage
+open ErrorUtils
+open Api
+
+(* You can swap the RPC engine, by using a different monad here,
+   note however that if you are using an asynchronous one, like
+   lwt or async, you should also use their specific IO functions
+   including the print functions.
+
+   You can easily put [ExnM] here and the code would stay unchanged.*)
+module M = Idl.IdM
+module IDL = Idl.Make(M)
+module Server = API(IDL.GenServer ())
+
+(** Command handler that runs Scilla with the
+    given [argv] and returns the resulting JSON. *)
+let runner argv =
+  let open IDL.ErrM in
+  (* Reset tracked libraries *)
+  GlobalConfig.StdlibTracker.reset ();
```

Comment:
I see this already called in `eval/Runner.ml:run`. I also added a suggestion to call it in `scilla_checker.ml`.
```suggestion
```

Reply:
Yes, you're right!